app/lib/log_util.rb

```diff
@@ -9,4 +9,8 @@ def self.log_err_and_airbrake(msg)
   def self.log_backtrace(exception)
     Rails.logger.error("Backtrace:\n\t#{exception.backtrace.join("\n\t")}")
   end
+
+  def self.log_warning(msg)
```

Comment:
nit: i dont think this justifies a method... Why not just use `Rails.logger.warn` directly? 
Is it because of the test?

Reply:
Nah, just following pattern. I agree, I'll change it